src/ufm/countermeasures.py

```diff
@@ -17,38 +17,22 @@ def forward(self, x):
         x = x + residual
         return x
 
-def run_countermeasures(model, config: dict, logger: logging.Logger):
+def run_countermeasures(model, method: str, logger: logging.Logger, **kwargs ):
```

Comment:
Why this change? I strongly think we should have the params in a config loaded by hydra.

Reply:
I wanted to decouple the code, but it might be simpler to pass the config itself.
I will revert the change.